src/components/NFTproduct/Nftproduct.tsx

```diff
@@ -176,9 +178,10 @@ const Nftproduct = () => {
           <BuyNow>
             <button
               onClick={() => {
+                setValue(false, auctionNFT?.auction.listingPrice || 0)
```

Comment:
This solution is work. 
it should be check and validate all data when page render.we handle and enhance it in another PR.
Thanks for your effort